opus/application/static_media/js/selectMetadata.js

```diff
@@ -17,8 +17,11 @@ var o_selectMetadata = {
     // will be used in mutation observer to determine if scrollbar location should
     // be set.
     isSortingHappening: false,
-    currentSelectedMetadata: opus.prefs.cols.slice(),
 
+    // save the original copy in case we need to discard
+    opusPrefsCols: opus.prefs.cols,
```

Comment:
I'm confused by this line. When this is executed, has opus.prefs.cols even been initialized properly yet? The statement will run at script load time. If opus.js was loaded first, then this will have the default columns. But the proper columns won't be set until the hash is parsed later during the initialization, so it seems like this would get the default columns by accident. The variable is set again during render(), so perhaps it would be less confusing just to set it to [] or null here?

Reply:
Also, wouldn't it be better to call it something like "originalOpusPrefsCols" since it isn't always equal to opus.prefs.cols?